### DISCUSSION TITLE
Astroid Framework 3.1.7 Bug with Firefox

### DISCUSSION THREAD

**Discussion by mmsarki** [templaza/astroid-framework#855] [General]
Hello,
With the latest version of the Astroid Framework it is not possible to scroll the page, it goes up by itself.
I've checked, it's the framework that's creating the problem, not the template.
Thank you for your work

**Comment by felixkat** [comment]
What do you mean "scroll the page"?

What template are you using?

**Comment by sonvnn** [comment]
@mmsarki The reason is the smooth-scrolling script. Because Polyscrolling has stopped maintaining their library. We have replaced PolyScroll with Lenis Scrolling. Since they are two different libraries, you need to Clear Astroid Cache and Server cache to ensure the new library is fully reloaded on your site.
